hummingbird/ml/ir_converters/topology.py

```diff
@@ -46,6 +47,10 @@ def convert(topology, device=None, extra_config={}):
     if operator_map[operators[-1].full_name].regression:
         # We are doing a regression task.
         pytorch_container = PyTorchBackendModelRegression
+    elif hasattr(operator_map[operators[-1].full_name], 'is_anomaly_detection') and \
```

Comment:
If you add `anomaly_detection` to `BaseOperator` (look at the comment below) you can remove the `hasattr`.

Reply:
will do!